### PR TITLE
WIP: Use kind 0.9.0

### PR DESCRIPTION
**Pull request by markxnelson**
Signed-off-by: Mark Nelson <mark.x.nelson@oracle.com>

# Use kind 0.9.0

This PR updates the pipeline to run on kind 0.9.0 instead of 0.7.0, use the right node image, and adapts to the new docker network requirements introduced in 0.9.0.  It also attempts to fix some tests that have been failing sporadically.  We suspected that the kind version/node image combination was causing slower startup time and that the tests were just not waiting long enough on some occassions.

Fixes VZ-1965

# How has this been tested?

Provide details about how you tested this changed, if necessary
provide instructions to reproduce the testing.

- [ ] Tested locally in my own environment
- [x] Successful acceptance test run on CI server
- [ ] Other (specify)

# Checklist 

As the author of this PR, I have:

- [x] Checked my code follows the style guidelines, ran `go fmt`
- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated godoc for all public functions
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [x] Added or updated acceptance tests if appropriate
- [x] Checked that I correctly spelled trademark and product names
- [x] Used inclusive language and neutral tone
- [x] Not included any sensitive information or hardcoded credentials

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: platform-operator/Makefile [hunk 1]
```diff
@@ -227,8 +227,10 @@ endif
 ifdef JENKINS_URL
 	# Get the ip address of the container running the kube apiserver
 	# and update the kubeconfig file to point to that address, instead of localhost
-	sed -i -e "s|127.0.0.1.*|`docker inspect ${CLUSTER_NAME}-control-plane | jq '.[].NetworkSettings.IPAddress' | sed 's/"//g'`:6443|g" ${KUBECONFIG}
+	sed -i -e "s|127.0.0.1.*|`docker inspect ${CLUSTER_NAME}-control-plane | jq '.[].NetworkSettings.Networks[].IPAddress' | sed 's/"//g'`:6443|g" ${KUBECONFIG}
 	cat ${KUBECONFIG} | grep server
+
+	$$(X=$$(docker inspect $$(docker ps | grep "jenkins-runner" | awk '{ print $$1 }') | jq '.[].NetworkSettings.Networks' | grep -q kind ; echo $$?); if [[ ! $$X -eq "0" ]]; then docker network connect kind $$(docker ps | grep "jenkins-runner" | awk '{ print $$1 }'); fi)
```
Comment (gilbode) [inline platform-operator/Makefile:233]: Why don't we do this in the Jeninsfile vs. the Makefile if it's a jenkins only thing?

File: Jenkinsfile [hunk 2]
```diff
@@ -11,7 +11,7 @@ pipeline {
 
     agent {
        docker {
-            image "${RUNNER_DOCKER_IMAGE}"
+            image "phx.ocir.io/stevengreenberginc/verrazzano/jenkins-runner:1.1.7"
```
Comment (pfmackin) [inline Jenkinsfile:14]: internal name

Comment (agv426) [reply Jenkinsfile:14]: maybe we can define EXPERIMENTAL_RUNNER_DOCKER_IMAGE so that test branches can drop that in instead.

**Review by agv426 — APPROVED** [review]

**Review by mgianatagh — APPROVED** [review]

File: platform-operator/test/integ/integ_test.go [hunk 3]
```diff
@@ -51,7 +51,7 @@ var _ = ginkgo.Describe("verrazzano-install namespace resources ", func() {
 		isPodRunningYet := func() bool {
 			return K8sClient.IsPodRunning(platformOperator, installNamespace)
 		}
-		gomega.Eventually(isPodRunningYet, "1m", "5s").Should(gomega.BeTrue(),
+		gomega.Eventually(isPodRunningYet, "2m", "5s").Should(gomega.BeTrue(),
```
Comment (mgianatagh) [inline platform-operator/test/integ/integ_test.go:54]: The comment on line 50 says "within 1m"

**Pull request merged**